src/search_datastructures.hpp

```diff
@@ -423,6 +407,12 @@ class GlobalDataHolder
     using TBlastRepFile = seqan::FormattedFile<seqan::BlastReport, seqan::Output, TIOContext>;
     using TBamFile      = seqan::FormattedFile<seqan::Bam, seqan::Output, seqan::BlastTabular>;
 
+    // Number of frames for subject and query sequences
+    static constexpr uint8_t qryNumFrames = (c_redAlph == AlphabetEnum::DNA3BS ? 4 :
+                                 (seqan::qIsTranslated(blastProgram) ? 6 : (seqan::qHasRevComp(blastProgram) ? 2 : 1)));
```

Comment:
```suggestion
    static constexpr uint8_t qryNumFrames = (c_redAlph == AlphabetEnum::DNA3BS)  ? 4 :
                                            seqan::qIsTranslated(blastProgram)   ? 6 : 
                                            seqan::qHasRevComp(blastProgram)     ? 2 : 1;
```

The ternary operator binds in such a way that very legible chaining is possible :)